pootle/checks.py

```diff
@@ -262,6 +262,50 @@ def check_settings(app_configs=None, **kwargs):
                     id="pootle.W015",
                 ))
 
+    if settings.POOTLE_TM_SERVER:
+        tm_indexes = []
+
+        if 'local' not in settings.POOTLE_TM_SERVER:
+            errors.append(checks.Critical(
+                _("POOTLE_TM_SERVER['local'] is missing.", server),
+                hint=_("Set POOTLE_TM_SERVER['local']."),
+                id="pootle.C008",
+            ))
+
+        for server in settings.POOTLE_TM_SERVER:
+            if 'INDEX_NAME' not in settings.POOTLE_TM_SERVER[server]:
+                errors.append(checks.Critical(
+                    _("POOTLE_TM_SERVER['%s'] has no INDEX_NAME.", server),
+                    hint=_("Set an INDEX_NAME for POOTLE_TM_SERVER['%s'].",
+                           server),
+                    id="pootle.C009",
+                ))
+            elif settings.POOTLE_TM_SERVER[server]['INDEX_NAME'] in tm_indexes:
+                errors.append(checks.Critical(
+                    _("Duplicate '%s' INDEX_NAME in POOTLE_TM_SERVER.",
+                      settings.POOTLE_TM_SERVER[server]['INDEX_NAME']),
+                    hint=_("Set different INDEX_NAME for all servers in "
+                           "POOTLE_TM_SERVER."),
+                    id="pootle.C010",
+                ))
+            else:
+                tm_indexes.append(settings.POOTLE_TM_SERVER[server]['INDEX_NAME'])
+
+            if 'WEIGHT' not in settings.POOTLE_TM_SERVER[server]:
+                errors.append(checks.Warning(
+                    _("POOTLE_TM_SERVER['%s'] has no WEIGHT.", server),
+                    hint=_("Set a WEIGHT for POOTLE_TM_SERVER['%s'].", server),
+                    id="pootle.W018",
+                ))
+            elif not (0.0 <= settings.POOTLE_TM_SERVER[server]['WEIGHT'] <= 1.0):
+                errors.append(checks.Critical(
```

Comment:
This isn't critical.  Rather set bounds where this is used in real life so that its always between 0.0-1.0.  Also this isn't critical its at best a Warning.  Critical means things stop working.  Stopping Pootle because the TM is slightly misconfigure isn't a good reason to not run Pootle.